File: src/Campaigns/Quick/send.js
import axios from 'axios';

const data = {
    title: "Announcement",
    sender: "VELSTACK",
    recipient: "0205550368",
    message: 'First API message from Velstack'
}

 axios.post('https://sms.velstack.com/api/messaging/quick/sms', data, {
    headers: {
        Authorization: "Bearer " + "API_KEY",
        Accept: "application/json",
    }
  }).then((response) => {
       alert('Message sent successfully')

  }).catch((errors) => {
      alert(errors.response.data.message)
  })

File: src/Campaigns/Quick/send.java


AsyncHttpClient client = new DefaultAsyncHttpClient();
 client.prepare("POST", "https://sms.velstack.com/api/messaging/quick/sms")
  .setHeader("accept", "application/json")
  .setHeader("Authorization", "Bearer API_KEY")
  .setBody("title", "Information")
    .setBody("sender", "velstack")
    .setBody("recipient", "0205550368")
    .setBody("message", "The message content")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

 client.close();